EasyPost/Shipment.cs

```diff
@@ -324,8 +309,6 @@ public static async Task<Shipment> Retrieve(string id)
             return await request.Execute<Shipment>();
         }
 
-        private static void FilterRates(ref List<Rate> rates, Func<Rate, bool> filter) => rates = rates.Where(filter).ToList();
```

Comment:
Hmm... what was this here for?

Also why removal?

Reply:
It was being used by the old `LowestRate` function when that function was inside/only used by the Shipment class.
Since that logic has been moved to a utility file, this is no longer needed.